frontend/src/components/layout/Header/Header.tsx

```diff
@@ -10,25 +10,40 @@ import {
   LogoLink,
   HeroContainer,
   HeroHeading,
+  LogoText,
+  MinimizedHeaderLink,
 } from './Header.styled';
 
-import { BlueLogo } from '../../logos';
+import { BlueBlackLogo, BlueLogo } from '../../logos';
 
-export const Header: React.FC = () => {
+interface HeaderProps {
+  readonly isFirstVisit: boolean;
+}
+
+export const Header: React.FC<HeaderProps> = ({ isFirstVisit }) => {
```

Comment:
Passing in a prop like this isn't the greatest approach.  It's almost an anti-pattern

Reply:
Yeah, it would be great to chat about this :)